### ISSUE TITLE
Error: this is not a registered game setting when rolling injury

### ISSUE THREAD

**Issue by scooper4711** [toastygm/HarnMaster-3-FoundryVTT#180]
HM 1.0.2 
Cannot roll an injury. When I press "determine injury", it tells me "error: this is not a registered game setting"

**Comment by scooper4711** [comment]
```foundry.js:3436 Uncaught (in promise) Error: This is not a registered game setting
    at ClientSettings.get (foundry.js:3436)
    at Function._calcInjury (dice-hm3.js:533)
    at Function.injuryRoll (dice-hm3.js:365)
    at Function._onChatCardAction (actor.js:992)
    at HTMLElement.dispatch (jquery.min.js:2)
    at HTMLElement.v.handle (jquery.min.js:2)```

**Comment by toastygm** [comment]
Thanks for the detailed information, I have fixed this and it will go out
in the next release.

Tom.

--
*Tom Rodriguez*
*w:* www.casualinsight.com | *e:* tmrodrig@gmail.com
*LinkedIn:* www.linkedin.com/in/tomrodriguez
*Twitter:* @TomRodriguezCI <https://twitter.com/TomRodriguezCI>

**Comment by toastygm** [comment]
fixed in 1.0.3
